CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## dbt 0.21.0 (Release TBD)
 
+### Fixes
+
+- Fix bigquery-specific synonym for schema (dataset) for snapshots. ([#3694](https://github.com/dbt-labs/dbt/issues/3694), [#xxxx](https://github.com/dbt-labs/dbt/pull/xxxx tba))
```

Comment:
It looks like there's a merge conflict with the changelog file, which happens when we've merged other PRs in the meantime. Could you pull the latest changes from `develop`, and then also add yourself to the list of Contributors? :)

Reply:
* fixed merge conflict from upstream.
* added PR ref number in changelog
* added myself to contributors
* set integration test config to use the alias 
Thanks Jeremy